.fixtures.yml

```diff
@@ -13,6 +13,7 @@ fixtures:
     foreman:       "git://github.com/theforeman/puppet-foreman"
```

Comment:
We should be able to get rid of a lot of the fixtures present here since they would no longer be referenced with the removal of foreman_proxy